docs/source/tags/magicwand.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: Magicwand
-type: tags
 order: 413
 meta_title: Magic Wand Tag for Quick Thresholded Flood Filling During Image Segmentation
 meta_description: Customize Label Studio with a Magic Wand tag to quickly click and drag to threshold flood fill image areas during image segmentation labeling for machine learning and data science projects.
+hide_sidebar: true
```

Comment:
Not sure if need it, just pointing it out because looks odd in this set of changes

Reply:
From my understanding, the entire Magic Wand tool is behind a FF. Rather than deleting a bunch of content from the code, I figured it would be cleaner to just hide the docs page.

Reply:
Update - Put the Magic Wand page back in after discussion with Andrew.